src/Apps/Artwork/Components/ArtworkSidebar/PrivateArtworkAdditionalInfo.tsx

```diff
@@ -0,0 +1,256 @@
+import * as DeprecatedAnalyticsSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import {
+  Clickable,
+  Flex,
+  FlexProps,
+  HTML,
+  Join,
+  ReadMore,
+  Spacer,
+  StackableBorderBox,
+  Text,
+} from "@artsy/palette"
+import { PrivateArtworkAdditionalInfo_artwork$key } from "__generated__/PrivateArtworkAdditionalInfo_artwork.graphql"
+import { useState } from "react"
+import * as React from "react"
+import { graphql, useFragment } from "react-relay"
+import { ArtworkDetailsMediumModalFragmentContainer } from "Apps/Artwork/Components/ArtworkDetailsMediumModal"
+import { useAnalyticsContext } from "System/Analytics/AnalyticsContext"
+import { ContextModule } from "@artsy/cohesion"
+import { useTracking } from "react-tracking"
+import { ConditionInfoModal } from "Apps/Artwork/Components/ArtworkDetails/ConditionInfoModal"
+import { RequestConditionReportQueryRenderer } from "Apps/Artwork/Components/ArtworkDetails/RequestConditionReport"
+import { PrivateArtworkDefinitionList } from "Apps/Artwork/Components/ArtworkSidebar/PrivateArtworkDefinitionList"
+
+// Number of items to display when read more is visible
+const COLLAPSED_COUNT = 3
+
+export interface PrivateArtworkAdditionalInfoProps extends FlexProps {
+  artwork: PrivateArtworkAdditionalInfo_artwork$key
+}
+
+export const PrivateArtworkAdditionalInfo: React.FC<PrivateArtworkAdditionalInfoProps> = ({
+  artwork,
+  ...flexProps
+}) => {
+  const data = useFragment(privateArtworkAdditionalInfoFragment, artwork)
+
+  const {
+    category,
+    series,
+    publisher,
+    manufacturer,
+    image_rights,
+    internalID,
+    canRequestLotConditionsReport,
+    signatureInfo,
+    conditionDescription,
+    certificateOfAuthenticity,
+    mediumType,
+    isUnlisted,
+  } = data
+
+  const [openMediumModal, setOpenMediumModal] = useState(false)
+  const [openConditionModal, setOpenConditionModal] = useState(false)
+
+  const { trackEvent } = useTracking()
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
+  const listItems = [
```

Comment:
I'm not sure just how much of this can be reused from the other component, but if there's an opportunity to share the code that would be good (on a follow-up PR if you want -- if you think its helpful!):

Thinking something like:

```
export const getArtworkAdditionalInfo = (artwork) => {
  const {
    category,
    series,
    publisher,
    manufacturer,
    image_rights,
    internalID,
    canRequestLotConditionsReport,
    signatureInfo,
    conditionDescription,
    certificateOfAuthenticity,
    mediumType,
    isUnlisted,
  } = artwork
  
  if (isUnlisted) {
    // do unlisted things 
  }

  const listItems = [ ... ] 
  
  return listItems 
}
```


And then you can import it in both components. 

```tsx
// in PrivateArtworkkAdditionalInfo.tsx

const listItems = getArtworkAdditionalInfo(artwork)

...
```

And the code is now shared, with less maintenance required.

Reply:
The truncation stuff fits nicely in an if but the list items are different for private works, would trying to push certain items to the list only for private works be kind of clunky?

Reply:
Its a matter of balancing things. Are there more items for private artworks than the norm? If so than a new component suffices. But if there are more shared items than not, then pushing things in isn't clunky, it just makes things easier to maintain in the future

Reply:
There are less items for private works, but we also want to keep the order consistent, so it would be a matter of pushing things in at certain places which feels like a headache. Basically I've decided if Peter wants "framed" in both I'll share them but if not I'll probably keep as is to avoid weird array splicing logic.

Reply:
Here's an example approach: https://github.com/artsy/force/pull/13961